### PR TITLE
Created QA.md and included a link to this MD file in README.md

### PR DESCRIPTION
**Pull request by sfrubio**

### CODE REVIEW

File: QA.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# QA code challenge ![alt Batlogo](public/images/batlogo-small.png)
+
+Batman needs your help! A supervillain was found.
+
+Interface created for Batdevices have to show information about the possible attacks to Gotham, but it is looking bugged...
```
Comment (rburrowes) [inline QA.md]: "The interface created for Batdevices should show information about the possible attacks in Gotham City, but it is looking buggy..."

Comment (rburrowes) [inline QA.md]: @sfrubio Did you mean to have no space between "Bat" and "devices"...?

File: QA.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+In this interface, Batman must be able to tell the Batcave's supercomputer the location (coordinate) of the recently found supervillain to identify which locations are most likely to be attacked.
```
Comment (rburrowes) [inline QA.md]: Suggested change, "....the location (coordinates) of the recently found supervillain to identify which locations are most likely to be attacked next time."

File: QA.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+You are requested to create acceptance tests scenarios to validate if coordinates sent by batmobile GPS are returning precise information about villains within the [Gotham boundaries](https://gist.githubusercontent.com/pitteri/d56780d610cb8e0a43bfa94fc54b71cd/raw/dcdd965c84cd05d856ae32646be69868d4a80afa/gotham_bbox.json).
```
Comment (rburrowes) [inline QA.md]: @sfrubio Are you suggesting that the candidate capture data from the "batmobile GPS" and test that the coordinates are true for the place of the attacks (i.e. the coordinates entered on the "batmobile GPS")?
I think I would say, "You are requested to create acceptance test scenarios to validate the data sent to the Batcave's supercomputer. 
The data must be:
- Come from an authenticated source
- Coordinates within the  [Gotham City Boundaries]
- etc.
"

File: QA.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+> Automated tests should be created by using using [Rest Assured](http://rest-assured.io/) or [Cucumber for Java](https://cucumber.io/)
+
+> Some good tutorials to follow: [Java Cucumber Tutorial](https://www.youtube.com/watch?v=pD4B839qfos&list=PL_noPv5wmuO_t6yYbPfjwhJFOOcio89tI) and [Rest Assured](http://www.baeldung.com/rest-assured-tutorial)
+
+> Remember, Batman is very demanding, even with the test scenarios you create. Bugs found are welcome. Surprise him!
```
Comment (rburrowes) [inline QA.md]: Suggested change, "Remember, Batman is a very demanding client, especially with the quality of his devices and the software they run. Bugs found are welcome. Confidence in his tools is a requirement. 
Surprise him!"

File: QA.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+Batman needs your help! A supervillain was found.
+
+Interface created for Batdevices have to show information about the possible attacks to Gotham, but it is looking bugged...
+
+In this interface, Batman must be able to tell the Batcave's supercomputer the location (coordinate) of the recently found supervillain to identify which locations are most likely to be attacked.
+
+You are requested to create acceptance tests scenarios to validate if coordinates sent by batmobile GPS are returning precise information about villains within the [Gotham boundaries](https://gist.githubusercontent.com/pitteri/d56780d610cb8e0a43bfa94fc54b71cd/raw/dcdd965c84cd05d856ae32646be69868d4a80afa/gotham_bbox.json).
+
+> Automated tests should be created by using using [Rest Assured](http://rest-assured.io/) or [Cucumber for Java](https://cucumber.io/)
```
Comment (rburrowes) [inline QA.md]: Suggested change, 
"Possible Automated tests tools 
- [Rest Assured](http://rest-assured.io/) 
- [Cucumber for Java](https://cucumber.io/)"

File: QA.md (hunk 1 above)
Comment (rburrowes) [inline QA.md]: Are there actual intentional bugs the Master branch code? Front-end, Back-end and Mobile???

Comment (sfrubio) [reply QA.md]: @rburrowes It is right. Any object that Batman own initiates with "Bat"
"To the Batmobile, let's go!" https://www.youtube.com/watch?v=pLMNxVDwUu8

File: QA.md (hunk 3 above)
Comment (sfrubio) [reply QA.md]: @rburrowes there is no authentication method in this case, so we should avoid "Come from an authenticated source".
I'm expected that the interviewee knows how to create test scenarios considering boundary-value analysis (in my opinion, the best technique to define test scenarios in this case)

File: QA.md (hunk 1 above)
Comment (sfrubio) [reply QA.md]: Not sure, let's give this work to interviewees ;-)

File: QA.md (hunk 3 above)
Comment (rburrowes) [reply QA.md]: It is possible for them to validate more than only the coordinates, right?

Comment (rburrowes) [reply QA.md]: I think that this description would be too leading and only is for the testing the batmobile interface. In effect, we are telling them that we only want this type of scenario. Coordinate validation. 
I would rather know if they can think outside of the obvious. If they can get something else from anything on the API. Are there any security holes? 
Maybe they would like to inject some SQL in parameters they think will go to a database. 
I have not yet read the readme files for the other parts of the system, but requirements can be gleaned from there to create scenarios.

Comment (rburrowes) [reply QA.md]: My edited suggestion,
"You are requested to create acceptance tests scenarios to validate the functionality of one or more of the systems developed.
* Batmobile Interface
* Batdevices Interface
* Batcave Supercomputer API
"

**Comment by rburrowes** [comment]
@robertomarin @jaimelopesflores This PR is ready for review. The QA team have completed the changes they would like to see.
